.github/workflows/coverage.yml

```
@@ -0,0 +1,38 @@
+name: Test Coverage
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)